Develop/assets/css/style.css

```diff
@@ -136,7 +169,11 @@ code {
 @media screen and (min-width: 768px) {
 
   /* on tablets, set columns to only be 50% of the width and not grow */
-  .card-column {}
+  .card-column .code-card {
+    /* new */
+    flex: 0 1 45%;
+
```

Comment:
Responsive design. Same as ln113 - May be better adding margin or other spacing code but 45% leaves room between code-cards.